packages/module/src/MessageDivider/MessageDivider.tsx

```diff
@@ -0,0 +1,35 @@
+// ============================================================================
+// Chatbot Main - Message Divider
+// ============================================================================
+import type { FunctionComponent } from 'react';
+import { Divider, Label } from '@patternfly/react-core';
+
+export interface MessageDividerProps {
+  /** Variant of the divider */
+  variant?: 'inset' | 'fullWidth';
+  /** Content of the message divider */
+  content?: string;
+}
+
+const MessageDivider: FunctionComponent<MessageDividerProps> = ({
+  variant = 'inset',
+  content = new Date().toLocaleDateString(),
+  ...props
+}: MessageDividerProps) => {
+  if (variant === 'inset') {
+    return (
+      <div className="pf-chatbot__message-divider pf-m-divider pf-m-wrap" {...props}>
```

Comment:
WDYT about the need for `.pf-m-wrap`? If the text should always wrap and never truncate, you could take the class off and move the `.pf-m-wrap` style block up into the top to L13 or so of `MessageDivider.scss`

Reply:
I think leaving it as a separate modifier may be fine for now, in case truncation is desired later (and so the modifier controlled by a flag)